client/src/components/feedback/send.js

```diff
@@ -0,0 +1,260 @@
+import React, { Component } from 'react';
+import { Button, Form } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import Api from '../../services/api';
+import '../../styles/feedback-send.css';
+import MyBadge from '../shared/my-batch';
+import CancelButton from '../shared/button/cancel';
+
+class FeedbackSend extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      description: '',
+      isLoading: false,
+      timerCount: 3,
+      showTimer: true,
+      tooShortFeedback: false,
+      tooLongFeedback: false,
+      feedbackSent: false,
+      feedbackNotSent: false,
+      inputDisabled: false,
+      attemptCount: false,
+      validInput: true,
+    };
+    this.onSubmit = this.onSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.tooShortFeedback = this.tooShortFeedback.bind(this);
+    this.tooLongFeedback = this.tooLongFeedback.bind(this);
+    this.sendApi = this.sendApi.bind(this);
+    this.cancel = this.cancel.bind(this);
+    this.showFeedbackSent = this.showFeedbackSent.bind(this);
+    this.showFeedbackNotSent = this.showFeedbackNotSent.bind(this);
+    this.showTimer = this.showTimer.bind(this);
+  }
+
+  onSubmit() {
+    if (this.state.validInput) {
+      if (this.state.description.length < 10) {
```

Comment:
Why feedback "Cool!" or "+1" aren't valid?

Reply:
I used 'BlaBlaCar' experience, where feedback can be no less than 10 symbols. It is thought that user should write such feedback that can be really like feedback, e.g. "I've celebrated my birthday here. Everything was prepared due to our time. Very good, I recommend".

Reply:
It isn't a good idea to limit clients in there rights. The better way is to make `description` field is required but let clients write what they want. Some people don't like to write a lot of text
Also our clients can evaluate feedback (set like or dislike)

Reply:
I agree with you. On the other hand, I read somewhere (sadly, can't find it now) that YouTube is indexing and giving attention to feedbacks containing 3 or more words. Yes, we are not YouTube, but feedback should have some context and 'information' for other people. And this can be reached by some kind of description. IMO, feedback should be not less that 10 symbols at least. 

Finaly, we can contact our requirement manager and check what is needed for this feature :)